--- .pio/libdeps/pico/ESP8266SdFat/src/SdFatConfig.h	2024-05-08 19:38:53.946524375 -0400
+++ patches/pico/SdFatConfig.h	2024-05-15 14:19:31.354690243 -0400
@@ -121,7 +121,7 @@
  * 3 - An external SPI driver derived from SdSpiBaseClass is always used.
  */
 #ifndef SPI_DRIVER_SELECT
-#define SPI_DRIVER_SELECT 0
+#define SPI_DRIVER_SELECT 3
 #endif  // SPI_DRIVER_SELECT
 /**
  * If USE_SPI_ARRAY_TRANSFER is one and the standard SPI library is
